File: tests/fixtures/openai-realtime/openai-realtime-transcription-model-choice-fixed/01-streaming-model.ts
const callerConfigMsg = {
  type: 'session.update',
  session: {
    modalities: ['text', 'audio'],
    input_audio_format: 'g711_ulaw',
    output_audio_format: 'g711_ulaw',
    input_audio_transcription: { model: 'gpt-realtime-whisper' },
    turn_detection: { type: 'server_vad' },
  },
};

export default callerConfigMsg;

File: tests/fixtures/openai-realtime/openai-realtime-transcription-model-choice-fixed/01_streaming_model.py
import json


async def configure_caller_session(ws):
    await ws.send(
        json.dumps(
            {
                "type": "session.update",
                "session": {
                    "input_audio_transcription": {"model": "gpt-realtime-whisper"},
                },
            }
        )
    )
